### DISCUSSION TITLE
[Terminal Error] error: unexpected argument '-m' found

### DISCUSSION THREAD

**Discussion by LareineHan** [Cyfrin/foundry-full-course-cu#863] [Q&A] [answered]
Hello, 
I am taking Lesson 7, and I got an error on my bash / zsh terminal when I run 

```zsh
forge test -m testFundFailsWithoutEnoughEth
```

```zsh
error: unexpected argument '-m' found

Usage: forge test [OPTIONS]
```
terminal worked as intended so far but now I suddenly got this error.
Why my command don't work as Patrick's? 
Test itself is fine and successful when I run 
```zsh
forge test
```
Thank you community.

**Comment by 0xusmanf** [comment, marked as answer]
`-m` command has been upgraded to `--mt`.

Use
```shell
forge test --mt testFundFailsWithoutEnoughEth
```
and it should work.

**Reply by LareineHan** [reply]
Thank you! You are awesome 🥇
